bot/cogs/watchchannels/talentpool.py

```diff
@@ -0,0 +1,233 @@
+import logging
+import textwrap
+from collections import ChainMap
+from typing import Union
+
+from aiohttp.client_exceptions import ClientResponseError
+from discord import Color, Embed, Member, User
+from discord.ext.commands import Context, group
+
+from bot.constants import Channels, Guild, Roles, Webhooks
+from bot.decorators import with_role
+from bot.pagination import LinePaginator
+from .watchchannel import WatchChannel, proxy_user
+
+log = logging.getLogger(__name__)
+STAFF_ROLES = Roles.owner, Roles.admin, Roles.moderator, Roles.helpers    # <- In constants after the merge?
+
+
+class TalentPool(WatchChannel):
+    """Relays messages of helper candidates to the talent-pool channel to observe them."""
```

Comment:
```suggestion
    """Relays messages of helper candidates to a channel to observe them."""
```

Reply:
Isn't the point of a `TalentPool` cog class to relay message to a talent pool channel instead of "a" channel?

Reply:
Yeah I was a bit torn. Saying "talent pool channel" feels redundant cause we already know we're reading a docstring for the talent pool. On the other hand "a channel" is too vague. On second though the former is fine.